src/photosList/usePhotoList.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+
+export const useFetchedPhotos = (userId) => {
+  const [photos, setPhotos] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    if (userId) {
+      fetch(`https://jsonplaceholder.typicode.com/albums?userId=${userId}`)
+        .then((response) => response.json())
+        .then((albums) =>
+          fetch(
+            `https://jsonplaceholder.typicode.com/photos?albumId=${albums[0].id}`,
+          ),
+        )
+        .then((response) => response.json())
+        .then((photos) => setPhotos(photos))
+        .catch((error) => {
+          console.log("error fetching data", error);
+        })
+        .finally(() => setIsLoading(false));
```

Comment:
Please create a an api file in `src/shared/api.js`

With following structure

```
function fetchAlbums(userId) {
  return fetch(`https://jsonplaceholder.typicode.com/albums?userId=${userId}`)
        .then((response) => response.json());
}


function fetchPhotos(albumId) {
  return  fetch(`https://jsonplaceholder.typicode.com/photos?albumId=${albums[0].id}`)
        .then((response) => response.json());
}

export const api = {
  fetchPhotos,
  fetchAlbums
}
```

Reply:
done